### DISCUSSION TITLE
Hasura are not working for me

### DISCUSSION THREAD

**Discussion by deprecated-guy** [hasura/graphql-engine#9768] [Question]
i run docker compose up -d  and go to localhost:8080, and do not watching graphql console in my browser

and i getting errors

hasura-postgres-1              | 
hasura-postgres-1              | PostgreSQL Database directory appears to contain a database; Skipping initialization
hasura-postgres-1              | 
hasura-postgres-1              | 
hasura-postgres-1              | 2023-07-10 18:24:42.299 UTC [1] FATAL:  database files are incompatible with server
hasura-postgres-1              | 2023-07-10 18:24:42.299 UTC [1] DETAIL:  The data directory was initialized by PostgreSQL version 12, which is not compatible with this version 15.3 (Debian 15.3-1.pgdg120+1).
hasura-postgres-1 exited with code 1

**Comment by SamirTalwar** [comment]
The problem is that you have a database existing in a Docker volume. The database was created with a different version of PostgreSQL and you haven't upgraded it.

You can either upgrade it, or if you don't care about the data, just stop everything and delete it:

```
$ docker compose down -v
```
